src/components/HURUmap/CategoryHeader/index.js

```diff
@@ -0,0 +1,45 @@
+import { Typography, Grid } from "@material-ui/core";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+const CategoryHeader = ({ title, description, image }) => {
+  const classes = useStyles();
+  return (
+    <div>
+      <Grid container alignItems="center">
+        {image && (
+          <div className={classes.image}>
+            <Image src={image} layout="fill" />
+          </div>
+        )}
+        {title && (
+          <Typography variant="h3" className={classes.title}>
+            {title}
+          </Typography>
+        )}
+      </Grid>
```

Comment:
```suggestion
      <Typography variant="h3" className={classes.title}>
        <div className={classes.image}>
          <Image src={image} layout="fill" />
        </div>
        {title}
      </Typography>
```

We can then set `display: "flex", alignItems: "center" in useStyles `to achieve the same effect without using `Grid`.

Reply:
Fixed